internal/query/user.go

```diff
@@ -825,13 +825,17 @@ func scanUser(row *sql.Row) (*User, error) {
 		&count,
 	)
 
-	if err != nil || count != 1 {
-		if errors.Is(err, sql.ErrNoRows) || count != 1 {
+	if err != nil {
+		if errors.Is(err, sql.ErrNoRows) {
 			return nil, zerrors.ThrowNotFound(err, "QUERY-Dfbg2", "Errors.User.NotFound")
 		}
 		return nil, zerrors.ThrowInternal(err, "QUERY-Bgah2", "Errors.Internal")
 	}
 
+	if count != 1 {
+		return nil, zerrors.ThrowNotFound(err, "QUERY-Dfbg2", "Errors.User.NotFound")
+	}
+
```

Comment:
This is actually a bug fixing.

When I was trying to make things to work, I was getting the following error:

```
Parent=(ERROR: relation \"projections.users10\" does not exist (SQLSTATE 42P01))" username=loginClient
```

That means the `error` is not `nil` and it is not `ErrNoRows`; but since the default value for `var count int` is `0` and that is not `1`; it will fall as a `Not Found` when in reality was an `Internal Error`

Reply:
The errors means that the table you referring to does not exist. Not the user itself. This ussualy happens when the tables or not yet setup correctly in the `zitadel setup` step. (which you tried to fix elsewhere.)

This change needs to be reverted.